projects/storefrontapp/src/environments/environment.ts

```diff
@@ -25,11 +25,12 @@ export const environment: Environment = {
   occApiPrefix: '/occ/v2/',
   cds: buildProcess.env.CX_CDS ?? false,
   b2b: buildProcess.env.CX_B2B ?? false,
-  cdc: buildProcess.env.CX_CDC ?? false,
+  cdc: buildProcess.env.CX_CDC ?? true,
```

Comment:
Is this change on purpose? If not, please don't forget to reverse it.

Reply:
noted, thank you